scripts/funcTests/runFuncTests.sh

```diff
@@ -22,13 +22,15 @@ mono --version
 # Download the CLI install script to cli
 echo "Installing dotnet CLI"
 mkdir -p cli
-curl -o cli/dotnet-install.sh -L https://dot.net/v1/dotnet-install.sh
+# Issue 8931 - DISABLED TEMPORARILY curl -o cli/dotnet-install.sh -L https://dot.net/v1/dotnet-install.sh
```

Comment:
This should be linking to an issue that tracks the revert of this change :)

Reply:
I have both. I just switched the comment to be the thing that undoes the change, not the thing that caused the change.